### ISSUE TITLE
peptide alpha-N-acetyltransferase activity children cause true path violation

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#22106]
peptide alpha-N-acetyltransferase activity
Molecular Function

Definition (GO:0004596 GONUTS page)
Catalysis of the reaction: acetyl-CoA + peptide = CoA + N-alpha-acetylpeptide. This reaction is the acetylation of the N-terminal amino acid residue of a peptide or protein.

has descendants:

GO:0008999    ribosomal-protein-alanine N-acetyltransferase activity | is_a
GO:1990190    peptide-glutamate-N-acetyltransferase activity | is_a
GO:1990189    peptide-serine-N-acetyltransferase activity

**Comment by hdrabkin** [comment]
It would seem 'GO:0034212. peptide N-acetyltransferase activity)  is a better parent for these three as it is not limited to N-terminal AA residue . 
That being said, the only time alanine , glutamate, or serine would have an "N-" to transfer the acetyl group do would be when they are at the N-terminus. 
OTOH, GO:0050176 phenylalanine N-acetyltransferase activity, which is a direct child of 'GO:0034212, would also fit the caveat (only time it would have a NH2- to be acetylated is if it were the terminal amino acid in a peptide.

Also, GO:0061733,peptide-lysine-N-acetyltransferase activity,  as defined,  has the epsilon NH2, but also sometimes if it is the N-terminal amino acid, would have the alpha NH2 also.

**Comment by ValWood** [comment]
So are the terms correct.

GO:1990190    peptide-glutamate-N-acetyltransferase activity
is only used by one. case in PomBAse:

UniProtKB:Q9UTI3 | ard1 | enables | GO:1990190    peptide-glutamate-N-acetyltransferase activity | ECO:0000314   IDA | PMID:23912279 |   | 284812 Schizosaccharomyces pombe (strain 972 / ATCC 24843) | PomBase
-- | -- | -- | -- | -- | -- | -- | -- | --

**Comment by ValWood** [comment]
GO:1990189    peptide-serine-N-acetyltransferase activity
has 4 EXP
1 PomBAse
1 UniProt
2 Ecocyc.

should we use something else?

**Comment by ValWood** [comment]
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3766382/
This paper does seem to say that serine is phosphorylated at the N-term  (presumably after the met is removed)

**Comment by hdrabkin** [comment]
AH good, I was just going to look at that paper. 
In each case, the reference must be consulted to see if in fact the substrate is in fact an N-terminal amino acid on a peptide.

I looked at one in MGI GO:1990189 peptide-serine-N-acetyltransferase activity and of the five annotations, 4 are IBAs from PAINT, and one is an ISO from human.

**Comment by ValWood** [comment]
This paper says:
NatA, which is composed of the catalytic NAA10 subunit and the auxiliary NAA15 subunit, is the most promiscuous of all NAT enzymes; it will traditionally acetylate an α-amino group on nascent peptide chains with an amino-terminal alanine, cysteine, glycine, serine, threonine, or valine residue1,22,24,25. Interestingly, recent studies demonstrate that NAA10 also exists as a monomer in cells and that it can acetylate the α-amino group of substrates with amino-terminal aspartic acid and glutamic acid residues that can be generated post-translationally, but will not acetylate traditional NatA substrates26

**Comment by hdrabkin** [comment]
SO, if anything, I would transfer the  GO:0050176 phenylalanine N-acetyltransferase activity, to a child of GO:0004596. Will  need to investigate the Lysine one. I wouldn't be surprised if it needs two terms.

**Comment by hdrabkin** [comment]
Since the enzyme is so promiscuous, I would question whether we need the individual terms as children (except for lysine). One could always use a has_input the specific the N-terminal residue amino acid.

**Comment by ValWood** [comment]
I agree, I don't think we need to be residue specific...

**Comment by hdrabkin** [comment]
I will add this to the next editors discuss

**Comment by hdrabkin** [comment]
Hi @ValWood 
I forgot to ask initially how the [peptide alpha-N-acetyltransferase activity children cause true path violation?

**Comment by ValWood** [comment]
I thought alpha was limited to the N-terminal residue.  Is that not right?

It seems to be what you imply also in
https://github.com/geneontology/go-ontology/issues/22106#issuecomment-914333361
and other comments?

**Comment by hdrabkin** [comment]
Yes, but if there is a term for lysine that isn't alpha specific, then might need two to cover both targets. 
However, more importantly can you please explain what the TPV actually is?

**Comment by hdrabkin** [comment]
But if they all say -N-acetyltransferase, aren't the all alpha-acetyltransferases (since, again, outside of lysine with both alpha (if N-terminal) and epsilon), any other amino acid's NH2 group has to be alpha. So these seem ok. 
One needs  peptide N-acetyltransferase activity to handle lysine's epsilon NH2.

**Comment by hdrabkin** [comment]
I'm thinking that the peptide lysine N-acetyltransferase activity may be a bit 'vague' The N could be either the alpha if the lysine is N-terminal , OR epsilon , which I think is the case for the histone substrates (child term).

**Comment by hdrabkin** [comment]
Great; I just quickly took a look at one of the papers annotated to GO:0008999  and the paper title specifes N-alpha acetyltranfases and another specifying 'N-terminal acetylation' so I'll go ahead
